setup.py

```diff
@@ -13,7 +13,7 @@
 ext_module = Extension('AoN',
                        [whole_path],
                        include_dirs=[np.get_include()])
-if __name__ == "__main__":
+if __name__ == "_  _main__":
```

Comment:
Extra space?

Reply:
yup. Good catch!  Not sure how it passed...